scripts/setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Set up script for dependencies and required libraries
+# 1. Install libfreetype6 libcairo2
+# 2. Install qahirah (cairo) wraper and dependencies
+# 3. Install other required python libraries
+
+apt-get -y update
+apt-get -y install libfreetype6 libcairo2 libsm6 libxext6 libfontconfig1 libxrender1 fontconfig
+
```

Comment:
add
mkdir -p qahirah
mkdir -p pybidi
mkdir -p python_freetype

Reply:
Or maybe it is better to add `git submodule update --init --recursive`